automation/deploy-bin.ts

```diff
@@ -15,70 +15,131 @@
  * limitations under the License.
  */
 import * as archiver from 'archiver';
-import * as Promise from 'bluebird';
+import * as Bluebird from 'bluebird';
 import * as fs from 'fs-extra';
+import * as _ from 'lodash';
 import * as mkdirp from 'mkdirp';
-import * as os from 'os';
 import * as path from 'path';
 import * as publishRelease from 'publish-release';
 
-import * as packageJSON from '../package.json';
-
-const publishReleaseAsync = Promise.promisify(publishRelease);
-const mkdirpAsync = Promise.promisify<string | null, string>(mkdirp);
-
+const mkdirpAsync = Bluebird.promisify<string | null, string>(mkdirp);
```

Comment:
`mkdirp-promise` also exists.

Reply:
All right Hedley, you win! I didn't even need to import `mkdirp-promise`, as I realised that `fs-extra` also provides `fs.mkdirp` with typing information and all, and it's already being imported in this same module... :-)